DSA-Campaign-Uplift-Estimation/src/main/java/com/google/sps/servlets/KeywordCampaign.java

```diff
@@ -14,21 +14,22 @@
 
 package com.google.sps.servlets;
 
-import java.io.IOException;
-import java.io.PrintWriter;
-import javax.servlet.annotation.WebServlet;
-import javax.servlet.http.HttpServlet;
-import javax.servlet.http.HttpServletRequest;
-import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
 
-@WebServlet("/")
-public class EmptyServlet extends HttpServlet {
+public class KeywordCampaign {
```

Comment:
Please comment classes and functions that aren't trivial. For example, here you may want to comment on what this class is for.

Reply:
The empty servlet class was deleted - that class had no purpose, it was just a blank page to copy the format of servlets off of. I thought the Keyword Campaign class is trivial, because it simply contains a keyword campaign object?

Reply:
Yeah, wasn't talking about the empty servlet class. But it's generally useful to comment what the purpose of the Keyword Campaign class is, since it isn't clear to someone who may not have as much familiarity with the code.

Reply:
Can you mention something like: "This represents the settings of the keyword campaign that can be copied to a DSA Campaign"